src/options/options.ts

```diff
@@ -1,7 +1,7 @@
 import * as Mustache from "mustache";
 import { Swagger } from "../swagger/Swagger";
 
-export interface TemplateLocations {
+export interface OptionTemplates {
```

Comment:
>OptionTemplates [](start = 17, length = 15)

Hm, TemplateOptions sounds a bit more natural.
Actually, should we just name this type Template?

Reply:
We could do that, there are some types already defined as `Template` or `Templates`. So wanted to avoid the name collision, but it would be nice to change it everywhere to be more consistent. 

I noticed that there were some parameters in some places names `templateLocations` which also would be nice to rename.

Reply:
Would love to see a cleanup of those names overall, its a bit messy for sure. Do you want to give it a shot? (If not that can be done later)

Reply:
I will take a look later today.

Reply:
Awesome, thank you!

Reply:
I renamed it, can't find any more places where "Template Locations" are used.